src/sass/app/components/market.scss

```diff
@@ -327,7 +343,117 @@
+                                        padding-left: 5px;
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+                .subgroup {
+                    display: flex;
+                    flex-direction: column;
+                    flex: 0 0 auto;
+                    padding-bottom: 40px;
+
+                    &:last-of-type {
+                        padding-bottom: 80px;
+                    }
+                    .label {
+                        font-size: 18px;
+                        padding: 10px 0px;
+                        font-weight: 700;
+                        flex: 1 0 auto;
+                        color: $COLOR_BLUE;
+
+                        &.sticky {
+                            position: absolute;
+                            width: 431px;
+                            top: 60px;
+                            background: $COLOR_WHITE;
+                            @media (max-width: 767px) {
+                                top: 123px;
+                                width: calc(100% - 18px);
+                            }
+                        }
+                        &.put_under {
+                            z-index: -1;
+                        }
+                    }
+                    .subgroup-name {
+                        font-size: 18px;
+                        padding: 10px 0;
+                        font-weight: 400;
+                        flex: 1 0 auto;
+                        color: $COLOR_BLUE;
+                        transition: none;
+                        @media (min-width: 320px) and (max-width: 767px) {
+                            font-size: 16px;
+                        }
+
+                        @media (min-width: 767px) {
+                            &.sticky {
+                                position: absolute;
+                                width: 431px;
+                                top: 60px;
+                                background: $COLOR_WHITE;
+                                @media (max-width: 767px) {
+                                    top: 123px;
+                                    width: calc(100% - 16px);
+                                }
+                            }
+                            &.put_under {
+                                z-index: -1;
+                            }
+                        }
+                    }
+                    .submarket {
+                        display: flex;
+                        flex-direction: column;
+                        flex: 1 0 auto;
+                        margin-bottom: 10px;
+
+                        .submarket_name {
+                            font-size: 12px;
+                            padding: 10px 0 0;
+                            flex: 1 0 auto;
+                            font-weight: 300;
+                            color: $COLOR_ORANGE;
```

Comment:
this could also relate to the mixin I mentioned for `.subgroup-name` since they're quite similar